src/Fmt_ast.ml

```diff
@@ -906,8 +906,8 @@ and fmt_pattern c ?pro ?parens ({ctx= ctx0; ast= pat} as xpat) =
           $ fmt_or_k last p.sep_after_final p.sep_after_non_final
         in
         hvbox 0
-          (Cmts.fmt c ppat_loc
-             (p.box
+          (p.box
+             (Cmts.fmt c ppat_loc
```

Comment:
This patch cause an unstable comment in `test/passing/comments.ml`.

Reply:
FYI, this commit is separated out in https://github.com/ocaml-ppx/ocamlformat/pull/1141

Do you know if that test was stable before https://github.com/ocaml-ppx/ocamlformat/pull/866 ? I don't know if there is some intended interaction between the other changes in that PR and this one, or if this change was just added to that PR by mistake.

Reply:
I think before that PR, it was a dropped comment. The test is added by this PR.

Reply:
Ok. The high-level question is if, knowing the details of 866, you know of a way to not drop this comment without the regression on the example in https://github.com/ocaml-ppx/ocamlformat/issues/1140 ?